packages/runtime/src/useCases/transport/relationships/AcceptRelationship.ts

```diff
@@ -25,23 +25,8 @@ export class AcceptRelationshipUseCase extends UseCase<AcceptRelationshipRequest
     }
 
     protected async executeInternal(request: AcceptRelationshipRequest): Promise<Result<RelationshipDTO>> {
-        const relationship = await this.relationshipsController.getRelationship(CoreId.from(request.relationshipId));
-        if (!relationship) {
-            return Result.fail(RuntimeErrors.general.recordNotFound(Relationship));
-        }
-
-        if (!relationship.cache) {
-            return Result.fail(RuntimeErrors.general.cacheEmpty(Relationship, relationship.id.toString()));
-        }
-
-        if (relationship.status !== RelationshipStatus.Pending) {
-            return Result.fail(RuntimeErrors.relationships.wrongRelationshipStatus(relationship.id.toString(), relationship.status));
-        }
-
-        const updatedRelationship = await this.relationshipsController.accept(relationship.id);
-
+        const updatedRelationship = await this.relationshipsController.accept(CoreId.from(request.relationshipId));
```

Comment:
I talked about not checking for the status. Please re-add recordNotFound and checkCache

Reply:
your reasoning is valid for those checks as well

Reply:
no, this is a refactoring and we don't do refactorings in such PRs. When we decide to remove such checks we have to remove them anywere. It's different with your new checks, as they were added in this PR.